persistence/mongo/src/main/scala/za/co/absa/spline/persistence/mongo/dao/BaselineLineageDAO.scala

```diff
@@ -46,6 +47,7 @@ abstract class BaselineLineageDAO extends VersionedLineageDAO with Logging {
   protected lazy val subComponents: Seq[SubComponent] = SubComponent.values
 
   protected lazy val dataLineageCollection: DBCollection = getMongoCollectionForComponent(Component.Root)
+  protected lazy val eventCollection: DBCollection = getMongoCollectionForComponent(Component.Event)
```

Comment:
I can move `eventCollection` into `LineageDaoV5` along with implementaion of methods like `getLineagesByPathAndInterval` and instead provide just dummy implementation so that MultiVersionDao can stay to be version-non-aware.